src/main/java/com/snowflake/kafka/connector/SnowflakeSinkConnectorConfig.java

```diff
@@ -112,6 +113,16 @@ public class SnowflakeSinkConnectorConfig {
   public static final String INGESTION_METHOD_DEFAULT_SNOWPIPE =
       IngestionMethodConfig.SNOWPIPE.toString();
 
+  // This is the streaming bdec file type which can be defined in config
+  public static final String SNOWPIPE_STREAMING_FILE_TYPE = "snowflake.streaming.file.type";
```

Comment:
could we use `file.version` instead of `file.type`?

Reply:
yes, let me change it